### ISSUE TITLE
显存释放

### ISSUE THREAD

**Issue by turkeymz** [PaddlePaddle/PaddleNLP#578]
训练完的时候，我想立即进行预测。发现32G显存已经不够了。
这里面是否大部分是模型训练时的运行参数？
如果是，建议增加一个显存释放接口，把模型训练时的运行参数释放掉。然后通过加载模型重新获取最新的参数进行预测

案例：『NLP打卡营』实践课5：文本情感分析
参数如下：
# 批量数据大小
batch_size = 32
# 文本序列最大长度
max_seq_length = 256
# 训练轮次
epochs = 10
# 训练过程中保存模型参数的文件夹
ckpt_dir = "skep_ckpt"
# len(train_data_loader)一轮训练所需要的step数
num_training_steps = len(train_data_loader) * epochs
# Adam优化器
optimizer = paddle.optimizer.AdamW(
    learning_rate=2e-5,
    parameters=model.parameters())
# 交叉熵损失函数
criterion = paddle.nn.loss.CrossEntropyLoss()
# accuracy评价指标
metric = paddle.metric.Accuracy()

**Comment by turkeymz** [comment]
使用数据集为ChnSentiCorp

**Comment by Steffy-zxf** [comment]
@turkeymz 你好！你可以根据实际的显存大小调整batch_size 和 max_seq_len 参数设置。另外预测时遇到显存不足的问题时，可以通过重启aitsudio项目，之后跳过模型训练部分，预测时直接加载之前保存的模型参数进行预测即可。

**Comment by libertatis** [comment]
这是notebook的问题，你不主动关闭服务，除非运行出错，它是不会释放资源的。所以特别讨厌实验有师弟师妹用开notebook做实验！嗯，你可以把代码改一下，把训练和预测分开。训练结束后，重启AIStudio，再进行预测~

**Comment by turkeymz** [comment]
关闭
